src/globus_cli/commands/group/set_policies.py

```diff
@@ -0,0 +1,109 @@
+from typing import Optional
+
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import CommaDelimitedList, command
+from globus_cli.termio import formatted_print
+
+from ._common import MEMBERSHIP_FIELDS, group_id_arg
+
+SIGNUP_FIELDS = [
+    "address",
+    "address1",
+    "address2",
+    "city",
+    "country",
+    "current_project_name",
+    "department",
+    "field_of_science",
+    "institution",
+    "phone",
+    "state",
+    "zip",
+]
```

Comment:
I think we can remove this, as there's now `MEMBERSHIP_FIELDS` instead, for the same purpose.

```suggestion
```

Reply:
Oh whoops, good catch